Jenkinsfile

```diff
@@ -20,6 +20,12 @@ pipeline {
                 sh 'bash -c "tools/reloc_spec_check.sh"'
             }
         }
+        stage('Install Python dependencies') {
+            steps {
+                echo 'Installing Python dependencies'
+                sh 'python3 -m pip install -r requirements.txt'
+            }
```

Comment:
I wish we could just subrepo it instead, this basically counters all new timesave.

Reply:
The package is written in C, so people would require to build it themselves. They would require to install the python dev package to build it too. People on Windows (cygwin, not wsl) would require the VisualStudio toolchain just to build it. Too much trouble imo.

Also, this step only adds 2 extra seconds on the jenkins side

Reply:
Yeah subrepo doesn't really help since Jenkins would have to build it anyways, which is almost certainly going to be longer than just installing a few python packages.